src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
```

Comment:
Consider adding an `aria-label` to the `<a>` tag for better accessibility, especially if the logo is the only content inside the link. This helps screen readers understand the purpose of the link.